packages/publisher/test/generate-packages.test.ts

```diff
@@ -114,6 +116,13 @@ testo({
       ),
     );
   },
+  readmeOnePeer() {
+    const dt = defaultFS();
+    const typing = new TypingsData(dt.fs, createRawPackage(License.Apache20), /*isLatest*/ true);
+    expect(createReadme(typing, dt.pkgFS("jquery"))).toEqual(
+      expect.stringContaining("Peer dependencies: [@types/express](https://npmjs.com/package/@types/express)"),
```

Comment:
is this the most obvious expect syntax for `stringContaining`? I'd *expect* something like `expect(createReadme(...)).stringContaining("...")`

The two things that weird me out are
1. toEqual, but then it's not actually testing full equality.
2. expect.stringContaining, which seems like namespacing, when all the other uses of `expect` are callable, meant to be read like a sentence.

This probably *is* the most direct way to write this, and `expect` syntax is just intentionally awful.

Reply:
Probably it'd be better to snapshot test this

Reply:
(I just copied from the test above)

Reply:
Yeah, all of the tests are like this, it's just how it's done: https://jestjs.io/docs/expect#expectstringcontainingstring

Weird...